### ISSUE TITLE
Pagination in search endpoint

### ISSUE THREAD

**Issue by MartinMouritzen** [Podcastindex-org/docs-api#92]
For the search endpoint, I would love a pagination option.

Right now to create eg. 10 pages of results, I have to request 100 results. It would be nice if I could add a param "offset" or something like that.

Hopefully it should be relatively easy to do in your PHP/SQL

$offset = intval($_REQUEST["offset"]) ? intval($_REQUEST["offset"]) : 0;

SELECT
blah
FROM
search
LIMIT ".$offset.",".$max."

Done deal! (yes, I know it's likely not as simple as that) 😄

**Comment by daveajones** [comment]
Is this "search/byterm" you are speaking of?

**Comment by MartinMouritzen** [comment]
Indeed. I am assuming it would also make sense for a few other endpoints
like the "trending" one.

**Comment by manhphim** [comment]
I agree. Currently I don't see a way that I can get a paginated response from the API

**Comment by marcmaxson** [comment]
In testing `/search/byterm/` today: I'm getting 60 feeds returned regardless of what I search for. I know that these searches should return more than 60. Is there a limit somewhere on search results?

**Comment by mainrs** [comment]
> In testing `/search/byterm/` today: I'm getting 60 feeds returned regardless of what I search for. I know that these searches should return more than 60. Is there a limit somewhere on search results?

There is a 60 item limit on search. I experience the same behavior when searching for popular terms. There is sadly no way to increase the maximum amount as well. Although this is more of a "hot fix". |

My use-case is showing users trending podcasts for a given category as well as "infinite scrolling" for both trending and search.

I've read multiple times that time is a problem for the pagination feature. Given that, I'd be willing to spend some time to implement this. You can contact me using my public GitHub E-Mail!
